src/t.jl

```diff
@@ -71,18 +71,17 @@ function show_params(io::IO, x::OneSampleTTest, ident="")
 end
 
 """
-    OneSampleTTest(xbar::Real, stddev::Real, n::Int, μ0::Real = 0)
+    OneSampleTTest(xbar::Real, stddev::Real, n::Int, μ0::Real = 0, df::Int = n-1)
```

Comment:
The docstring should mention `df`. Probably better call it `dof` by the way, since it's the name of the function in StatsBase.

Reply:
I've changed df to dof all within t.jl, but I'm concerned that this is a breaking change.